test/letters/containers/VeteranBenefitSummaryLetter.unit.spec.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import { VeteranBenefitSummaryLetter } from '../../../src/js/letters/containers/VeteranBenefitSummaryLetter';
+
+const defaultProps = {
+  benefitSummaryOptions: {
+    benefitInfo: {
+      awardEffectiveDate: '1965-01-01T05:00:00.000+00:00',
```

Comment:
These date value trigger a moment() warning in unit tests, though they still pass. Leaving them as is (copied from a vets-api response) so we can remember to figure out if the values returned from EVSS need to be formatted or parsed differently on the front-end.

```
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format.
moment construction falls back to js Date(), which is not reliable across all browsers and versions.
Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release.
Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
```